pkg/sidecar/sidecar.go

```diff
@@ -167,10 +171,19 @@ func (s *Sidecar) setupClients(ctx context.Context) error {
 // updateCertificates Updates the certificates stored in disk and signal the Process to restart
 func (s *Sidecar) updateCertificates(svidResponse *workloadapi.X509Context) {
 	s.config.Log.Debug("Updating X.509 certificates")
+	svidFile := path.Join(s.config.CertDir, s.config.SVIDFileName)
+	svidKeyFile := path.Join(s.config.CertDir, s.config.SVIDKeyFileName)
+	svidBundleFile := path.Join(s.config.CertDir, s.config.SVIDBundleFileName)
 	if err := disk.WriteX509Context(svidResponse, s.config.AddIntermediatesToBundle, s.config.IncludeFederatedDomains, s.config.CertDir, s.config.SVIDFileName, s.config.SVIDKeyFileName, s.config.SVIDBundleFileName, s.config.CertFileMode, s.config.KeyFileMode); err != nil {
 		s.config.Log.WithError(err).Error("Unable to dump bundle")
+		s.fileWritesSuccess[svidFile] = false
+		s.fileWritesSuccess[svidKeyFile] = false
+		s.fileWritesSuccess[svidBundleFile] = false
 		return
 	}
+	s.fileWritesSuccess[svidFile] = true
+	s.fileWritesSuccess[svidKeyFile] = true
+	s.fileWritesSuccess[svidBundleFile] = true
```

Comment:
Everything here is either all true or all false. I think we can just consolidate to a single variable like `x509WriteSuccess`

Reply:
Are you proposing the JSON returned would no longer be a map of filenames to boolean, but instead something like
```json
{
  x509WriteSuccess: true/false
  jwtWriteStatus: {
    file1.jwt: true/false
    file2.jwt: true/false
  }
}
```
?